DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: affirm
 Title: Secular affirmations against data
-Version: 0.2.0.9001
+Version: 0.2.0.9002
```

Comment:
```suggestion
Version: 0.2.0.9001
```

If we increment the dev version here, then that dev version should also be referenced in news. For simplicity, let's keep the version the same.